docs/api/address/get-address-transactions.schema.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "GET request that returns account transactions",
+  "title": "AccountTransactions",
+  "type": "object",
+  "required": ["results", "limit", "offset", "total"],
+  "properties": {
+    "limit": {
+      "type": "integer",
+      "maximum": 30
+    },
+    "offset": {
+      "type": "integer"
+    },
+    "total": {
+      "type": "integer"
+    },
+    "results": {
+      "type": "array",
+      "items": {
+        "$ref": "../../entities/transactions/abstract-transaction.schema.json"
```

Comment:
This only returns the base set of abstracts props, not all transactions?

If it returns mempool and confirmed txs, we could write as follows, which would result in nicer compiled type output, 

```json
    "results": {
      "type": "array",
      "items": {
        "type": "object",
        "oneOf": [
          {
            "$ref": "<path to mempooltx>"
            {
          },
            "$ref": "<path to tx>"
          },
        ]
```

Reply:
Great suggestion, thanks!